lib/instrument/headers.js

```diff
@@ -0,0 +1,260 @@
+import {archive, unarchive} from './transformer/nskeyed';
+import plistlib from 'bplist-parser';
```

Comment:
npm modules cannot be used unless they are defined as depencies in package.json
Also, all plist-related stuff could be found in the `@appium/support` lib. Check https://github.com/appium/appium/blob/2.0/packages/support/test/plist-specs.js and other examples from this module for more details